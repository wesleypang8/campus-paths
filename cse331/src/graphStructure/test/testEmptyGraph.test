#Test construct an empty graph

#Create a graph
CreateGraph emptyG

#List nodes 
ListNodes emptyG