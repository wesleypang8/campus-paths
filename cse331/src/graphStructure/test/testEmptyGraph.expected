#Test construct an empty graph

#Create a graph
created graph emptyG

#List nodes 
emptyG contains: